crontab

```diff
@@ -1,9 +1,10 @@
-SHELL=/bin/bash
+cSHELL=/bin/bash
```

Comment:
Is this an intended change or a typo? If it's intended, what does it do?

Reply:
oops good catch! will fix

Reply:
fixed